docs/guides/jsdom.md

```diff
@@ -17,7 +17,7 @@ As a result, a standalone script like the one below is generally a good approach
 var jsdom = require('jsdom').jsdom;
 
 global.document = jsdom('');
-global.window = document.defaultView;
+global.window = global;
 Object.keys(document.defaultView).forEach((property) => {
```

Comment:
Yes, node indeed has `setTimeout`.

Also, the point of `jsdom` is to provide all the browser globals to `node`, so as to fake that it's a browser.